root/etc/e-smith/events/actions/nethserver-ipsec-tunnels-edit

```diff
@@ -21,16 +22,29 @@
 event=$1
 action=$2
 tunnel="${3}_ipsec-tunnel"
+remoteIP=$4
 
 if [ -z "$event" ]; then
     exit 0
 fi
 
 if [ "$action" == "create" ]; then
-    ipsec auto --start "$tunnel"
+    /usr/sbin/ipsec auto --start "$tunnel"
 elif [ "$action" == "modify" ]; then
-    ipsec auto --delete "$tunnel"
-    ipsec auto --start "$tunnel"
-elif [ "$action" == "delete" ]; then
-    ipsec auto --delete "$tunnel"
+    /usr/sbin/ipsec auto --delete "$tunnel"
+    /usr/sbin/ipsec auto --start "$tunnel"
+elif [ "$action" == "delete" ] || [ "$action" == "disable" ]; then
+    /usr/sbin/ipsec auto --delete "$tunnel"
+
+    # Destroy a non existing connexion with conntrack leads to stderr error
+    TEST500=$(/usr/sbin/conntrack -L  -d  $remoteIP --proto UDP --sport 500 --dport 500)
+    if [[ $TEST500 != '' ]];then
+        /usr/sbin/conntrack -D  -d  $remoteIP --proto UDP --sport 500 --dport 500
+    fi
+
+    TEST4500=$(/usr/sbin/conntrack -L  -d  $remoteIP --proto UDP --sport 4500 --dport 4500)
+    if [[ $TEST4500 != '' ]];then
+        /usr/sbin/conntrack -D  -d  $remoteIP --proto UDP --sport 4500 --dport 4500
+    fi
+
```

Comment:
The output on stderr should be there even on success, so we can just ignore it and use
the exit code from delete command,

```suggestion
    /usr/sbin/ipsec auto --delete "$tunnel"
    ret=$?

    if [ ! -z "$remoteIP" ]; then
        /usr/sbin/conntrack -D  -d  $remoteIP --proto UDP --sport 500 --dport 500 2>/dev/null
        /usr/sbin/conntrack -D  -d  $remoteIP --proto UDP --sport 4500 --dport 4500 2>/dev/null
    fi
    
    exit $ret
```